### ISSUE TITLE
Invalid URL in manifest

### ISSUE THREAD

**Issue by cschol** [Jojosito/MUS-X#5]
```
[MUS-X] Validating plugin.json...         
Exception validating URL: mus-x.de (unknown url type: 'mus-x.de')
FAILED     
[MUS-X] Issues found in `plugin.json`:
                                                                             
Invalid URL: mus-x.de 
```

Please add `https://` prefix to the URL in the plugin manifest. Thank you.

**Comment by Jojosito** [comment]
Fixed in 081971e6b22dde600584f7ed772d423a974b6139

**Comment by cschol** [comment]
The URL is still invalid.
 
 ```
 [MUS-X] Validating plugin.json...                                                                                                                                                                                                                                                                                      
Exception validating URL: https://mus-x.de (<urlopen error [SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1007)>)                                                                                                                                                                   
FAILED                                                                                                                                                                                                                                                                                                                 
[MUS-X] Issues found in `plugin.json`:                                                                                                                                                                                                                                                                                 
                                                                                                                                                           
Invalid URL: https://mus-x.de       
```

![image](https://github.com/Jojosito/MUS-X/assets/624427/e129a4b9-afca-478e-93bf-d07529c73ce4)

Please provide a valid URL.
